src/components/fabSpeedDial/fabController.js

```diff
@@ -135,9 +135,13 @@
 
     // Fire the animations once in a separate digest loop to initialize them
     function fireInitialAnimations() {
-      $mdUtil.nextTick(function() {
-        $animate.addClass($element, 'md-noop');
-      });
+      var delay = 100; // Delay until after the element has been appened to the DOM
```

Comment:
@jelbourn Totally agree; let me try `$$rAF`, but the main issue is that I need the elements to be visible on-screen before the animations run because they do some calculations based on the spacing.

The **real** issue here is that the "default" non-animated state is for the element to be opened. We use the JS animations to close it. Ideally, we'd flip this so that the default state was closed and we opened it with the animations, but that would require rewriting it to not use flex positioning, which I'd like to do, but will take a while, so this is a temporary fix.

I'll let you know if `$$rAF` works or not.